test/runtests.jl

```diff
@@ -253,6 +274,15 @@ const DATADIR = joinpath(@__DIR__, "data")
         edf = EDF.File(io)
         @test sprint(show, edf) == "EDF.File with 140 16-bit-encoded signals"
     end
+
+    @testset "Exports readable by MNE" begin
+        edf = EDF.read(joinpath(DATADIR, "test_float_extrema.edf"))
+        @test edf.signals[1].header.digital_minimum ≈ -32767.0f0
+        edf = @set edf.signals[1].header.digital_minimum = -32767 * 2
```

Comment:
I don't understand what the purpose of this is. If it's necessary for testing a particular behavior, can you add a comment? I think it's the only use of Accessors here so if it's not actually necessary then dropping it would reduce the number of test dependencies.

Reply:
this PR was meant to complement https://github.com/beacon-biosignals/EDF.jl/pull/72 which uses Accessors more extensively in the tests

however I no longer remember what the point of this line is anymore :/